spec/models/authenticator/httpd_spec.rb

```diff
@@ -707,17 +719,55 @@ def authenticate
         let(:headers) do
           super().merge('X-Remote-User-Groups' => 'wibble@fqdn,bubble@fqdn')
         end
+
+        it "handles a comma separated grouplist" do
+          expect(subject).to receive(:find_external_identity).with(username, user_attrs, ["wibble@fqdn", "bubble@fqdn"])
+          authenticate
+        end
+      end
+
+      context "using custom delimiter in settings" do
+        let(:config) { {:httpd_role => true, :group_delimiters => ","} }
+        let(:headers) do
+          super().merge('X-Remote-User-Groups' => 'wibble:wobble@fqdn,bubble@fqdn')
+        end
+
+        it "handles a group with a colon" do
```

Comment:
```suggestion
        it "handles a group with a comma" do
```

Also the other `it`s below have the wrong term as well.